playbooks/06-deploy-architecture.yml

```diff
@@ -197,6 +197,15 @@
         - update_containers
         - edpm_bootstrap
 
+    - name: Update containers in deployed OSP operators using set_openstack_containers role
+      vars:
+        cifmw_set_openstack_containers_extra_vars: "{{ cifmw_edpm_prepare_extra_vars }}"
+      ansible.builtin.include_role:
+        name: set_openstack_containers
+      tags:
+        - set_openstack_containers
+        - edpm_bootstrap
+
```

Comment:
I see.  In the update job then you just add:

```yaml
      cifmw_reproducer_play_skip_tags:
        - update_containers
```
and you get the right type of deployment, but by default nothing change of other uni job, they still deploy the "hotfix" kind of deployment.

Reply:
cifmw_reproducer_play_skip_tags mechanism is removed after review from Pablo